src/eos-google-chrome

```diff
@@ -69,28 +69,9 @@ class GoogleChromeLauncher:
             launcher_process.wait()
             logging.info("Google Chrome launcher stopped")
 
-    def _run_app_center_for_chrome(self):
-        # We use the APP ID as the one for GNOME Software, to let it choose the best one.
-        chrome_app_center_id = self.FLATPAK_CHROME_APP_ID
-
-        # FIXME: Ideally, we should be able to pass 'com.google.Chrome' to GNOME Software
-        # and it would do the right thing by opening the page for the app's branch matching
-        # the default branch for the apps' source remote. Unfortunately, this is not the case
-        # at the moment and fixing it is non-trivial, so we'll construct the full unique ID
-        # that GNOME Software expects, right from here, based on the remote's metadata.
-        default_branch = None
-        try:
-            remote = self._installation.get_remote_by_name(self.FLATPAK_REMOTE_EOS_APPS)
-        except GLib.Error as e:
-            logging.warning("Could not determine default branch for remote %s", self.FLATPAK_REMOTE_EOS_APPS)
-
-        default_branch = remote.get_default_branch()
-        if default_branch:
-            chrome_app_center_id = 'system/flatpak/{}/desktop/{}.desktop/{}'.format(self.FLATPAK_REMOTE_EOS_APPS,
-                                                                                    self.FLATPAK_CHROME_APP_ID,
-                                                                                    default_branch)
+    def _install_chrome(self):
         try:
-            subprocess.Popen(['gnome-software', '--details={}'.format(chrome_app_center_id)])
+            subprocess.Popen(['eos-google-chrome-installer'])
```

Comment:
Getting architecture string is easy, I think I will just format this string with arch info.